dags/bqetl_main_summary.py

```diff
@@ -46,7 +51,12 @@
         dataset_id="telemetry_derived",
         project_id="moz-fx-data-shared-prod",
         owner="jklukas@mozilla.com",
-        email=["jklukas@mozilla.com"],
+        email=[
+            "dthorn@mozilla.com",
+            "frank@mozilla.com",
+            "jklukas@mozilla.com",
+            "telemetry-alerts@mozilla.com",
```

Comment:
Do we always want to email everyone listed under `email` for the DAG for each task? Just wondering, since that hasn't been the case in the past: https://github.com/mozilla/telemetry-airflow/blob/b872df04bad82e6e4c20c06d9404f95822920e49/dags/main_summary.py#L368

Reply:
no, but in those cases we should set `email` in `metadata.yaml` to reduce the list.

Reply:
Ah, I see.